doc/index.qdoc

```diff
@@ -1,68 +1,94 @@
+/****************************************************************************************
+**
+** Copyright (C) 2019 Jolla Ltd.
+** Contact: David Llewellyn-jones <david.llewellyn-jones@jollamobile.com>
+** All rights reserved.
+**
+****************************************************************************************/
+
 /*!
 \contentspage {Sailfish App Library}
-\page index.html
+\page index
```

Comment:
Why did you remove ".html"? Now QtC complains that it can't find index.html!

Reply:
Good question. I don't recall why I changed this, but probably for no good reason. I've changed it back to 'index.html'.